packages/fxa-auth-db-mysql/lib/db/mem.js

```diff
@@ -97,7 +97,7 @@ module.exports = function(log, error) {
       return P.reject(error.duplicate());
     }
 
-    accounts[uid.toString('hex')] = data;
+    accounts[uid.toString('hex')] = extend({}, data);
```

Comment:
Previously, we'd just store whatever object the calling code gave us, meaning that it could change unexpectedly.